daemon/api.go

```diff
@@ -1449,14 +1450,16 @@ func postCreateUser(c *Command, r *http.Request, user *auth.UserState) Response
 	}
 
 	gecos := fmt.Sprintf("%s,%s", createData.EMail, v.OpenIDIdentifier)
-	if err := osutilAddExtraSudoUser(v.Username, v.SSHKeys, gecos); err != nil {
+	if err := osutilAddExtraUser(v.Username, v.SSHKeys, gecos, createData.Sudoer); err != nil {
 		return BadRequest("cannot create user %s: %s", v.Username, err)
 	}
 
 	var createResponseData struct {
-		Username string `json:"username"`
+		Username    string `json:"username"`
+		SshKeyCount int    `json:"ssh_key_count"`
 	}
 	createResponseData.Username = v.Username
+	createResponseData.SshKeyCount = len(v.SSHKeys)
```

Comment:
Note this may actually be useful, as we can present exactly which keys were imported.